airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -2520,6 +2520,34 @@ definitions:
       type:
         type: string
         enum: [JsonlDecoder]
+  JsonSchemaPropertySelector:
```

Comment:
I have a brainfart: for source-hubspot, didn't we have to filter out some of the fields too to only keep the `property_*` ones of we actually need to query for root level fields like `id`?

Reply:
we don't need to query for fields like `id`, or `updated_at`. In the current code, we already filter out these types of fields because when we fetch the properties from the external endpoint, they won't contain things like `updated_at` so they're basically being filter out since we only return the interesection of the set of schema fields and the static/dynamic property fields